test/e2e/Tests/Helpers/HttpHelpers.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Durable.Tests.DotnetIsolatedE2E
+{
+    class HttpHelpers
+    {
+        public static async Task<HttpResponseMessage> InvokeHttpTrigger(string functionName, string queryString = "")
+        {
+            // Basic http request
+            HttpRequestMessage request = GetTestRequest(functionName, queryString);
```

Comment:
Same suggestion about disposing disposable objects to prevent resource leaks.
```suggestion
            using HttpRequestMessage request = GetTestRequest(functionName, queryString);
```